### PR TITLE
Fix gameview layout

### PR DESCRIPTION
**Pull request by cyrilgourgouillon**

### CODE REVIEW

File: src/components/Common/Button/Button.tsx [hunk 1]
```diff
@@ -9,6 +9,7 @@ interface ButtonSpec {
 	children: ReactNode,
 	onClick?: React.MouseEventHandler<HTMLButtonElement>,
 	disabled?: boolean,
+	fullHeigth?: boolean,
```
Comment (adrien-nf) [inline src/components/Common/Button/Button.tsx:12]: This is not how "height" is supposed to be written...

File: src/components/Common/SectionTitle/SectionTitle.tsx [hunk 2]
```diff
@@ -8,7 +8,7 @@ interface SectionTitleSpec {
 export default function SectionTitle(props: SectionTitleSpec): JSX.Element {
 
 	return (
-		<div className={'flex flex-col mt-6 mb-6 relative uppercase'}>
+		<div className={'flex flex-col relative uppercase'}>
```
Comment (adrien-nf) [inline src/components/Common/SectionTitle/SectionTitle.tsx:11]: Why between {} ?

File: src/components/Lobby/GameView/GameView.tsx [hunk 3]
```diff
@@ -170,65 +170,89 @@ export default function GameView(props: GameProps): JSX.Element {
...
-								onClick={() => setIsStartVoteModalVisible(true)}>
-								{t('gameView.startVote')}
-							</Button>
-						) : null
-					}
+				<div className='flex flex-col flex-shrink-0 ml-8 mr-8'>
+					<div className='flex flex-row justify-between'>
+						<div className='h-16'>
+							<Countdown limitDate={dayjs(props.game.limitDate)} onFinish={nextDrawing} />
+						</div>
```
Comment (adrien-nf) [inline src/components/Lobby/GameView/GameView.tsx:190]: Weird linebreak here

File: src/components/Lobby/GameView/GameView.tsx [hunk 4]
```diff
@@ -170,65 +170,89 @@ export default function GameView(props: GameProps): JSX.Element {
...
-								size='medium'
-								icon={faArrowRight}
-								disabled={!hasLost}
-								onClick={props.leaveGame}>
-								{t('lobbyView.leaveBtnLabel')}
-							</Button>
-						) : null
-					}
+				<div className='flex flex-col justify-between flex-1 w-52'>
+
```
Comment (adrien-nf) [inline src/components/Lobby/GameView/GameView.tsx:204]: Weird linebreak here

File: src/components/Lobby/GameView/GameView.tsx [hunk 5]
```diff
@@ -170,65 +170,89 @@ export default function GameView(props: GameProps): JSX.Element {
...
+					<div className='h-12'>
+						{
+							(!hasLost) ? (
```
Comment (adrien-nf) [inline src/components/Lobby/GameView/GameView.tsx:207]: Should be (hasLost) not (!hasLost)

**Review by adrien-nf — CHANGES_REQUESTED** [review]

File: src/components/Common/Button/Button.tsx (hunk 1 above)
Comment (cyrilgourgouillon) [reply src/components/Common/Button/Button.tsx:12]: well

**Review by adrien-nf — APPROVED** [review]

**Pull request merged**